src/status_im2/contexts/quo_preview/transaction_sheet/transaction_sheet.cljs

```diff
@@ -0,0 +1,91 @@
+(ns status-im2.contexts.quo-preview.transaction-sheet.transaction-sheet
+  (:require [quo2.components.tabs.account-selector :as acc-sel]
+            [utils.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im2.contexts.quo-preview.tabs.segmented-tab]
+            [quo2.components.tabs.segmented-tab :as quo2]
+            [status-im2.contexts.shell.jump-to.utils :as utils]
+            [quo2.components.inputs.locked-input.view :as locked-input]
+            [quo2.components.markdown.text :as text]))
+
+(let [{:keys [width]} (utils/dimensions)]
+  (def screen-width width))
+(def account-selector-width (* 0.895 screen-width))
+(def shared-selector-list-data
+  {:show-label?  false
+   :transparent? false
+   :style        {:width  account-selector-width
+                  :height 40}})
+(def unique-selector-list-data
+  [{:account-text  "Drakaris account"
+    :account-emoji "🔥"}
+   {:account-text  "Daenerys account"
+    :account-emoji "👸"}])
+(def selector-list-data
+  (map #(merge % shared-selector-list-data)
+       unique-selector-list-data))
+
+(def label-style
+  {:color          colors/neutral-50
+   :letter-spacing -0.039
+   :line-height    18.2
+   :font-weight    :normal
+   :font-size      13})
+
+(defn- render-account-selectors
+  [item]
+  [rn/view {:style {:margin-right 10}}
+   [rn/touchable-opacity {:on-press #(js/alert (str "Pressed " (item :account-text)))}
+    [acc-sel/account-selector item]]])
+(defn preview-transaction-sheet
+  []
+  [rn/view
+   {:background-color (colors/theme-colors colors/white colors/neutral-90)
+    :flex             1
+    :flex-direction   :column
+    :padding-top      20
+    :padding-left     20}
+   [text/text
+    {:style {:font-size      19
+             :letter-spacing -0.304
+             :font-weight    :600
```

Comment:
Please use defined constants for the font-size, letter-spacing & font-weight.

You can find it here: https://github.com/status-im/status-mobile/blob/0a8993bbf19fc368e4f3f04ed5daf73062712203/src/quo2/foundations/typography.cljs